lbrynet/core/client/ConnectionManager.py

```diff
@@ -220,8 +204,22 @@ def _connect_to_peer(self, peer):
 
         log.debug("%s Trying to connect to %s", self._get_log_name(), peer)
         factory = ClientProtocolFactory(peer, self.rate_limiter, self)
```

Comment:
we attach a callback to the connection_was_made deferred that calls _peer_disconnected() , now ClientProtocol no longer needs to call disconnect peers in the ConnectionManager. This is good, because we need to reduce and eventually remove ClientProtocol's dependency on ConnectionManager.